sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsBufferCatalog.scala

```diff
@@ -659,38 +642,19 @@ class RapidsBufferCatalog(
           logDebug(s"Skipping spilling $buffer ${buffer.id} to ${spillStore.name} as it is " +
             s"already stored in multiple tiers")
         }
+>>>>>>> 740d1175060555ad373f30ed780a58215b6c3419
```

Comment:
I don't think the upmerge was done properly.

Reply:
you are right, this method needs to go away (it's now in the store). Fixed in the latest commit.